### PR TITLE
Add Soviet 11a

### PR DESCRIPTION
**Pull request by Smittytron**
Standard destroy the enemy mission. KSK_Nico did most of the legwork here.

### CODE REVIEW

**Comment by darcyrush** [comment]
Hard plays ok

File: mods/ra/maps/soviet-11a/soviet11a-AI.lua [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+ProductionDelays =
+{
+	easy = 3,
+	normal = 2,
+	hard = 1
+}
+
+IdleHunt = function(unit) if not unit.IsDead then Trigger.OnIdle(unit, unit.Hunt) end end
+
+SendCrusier = function()
```
Comment (reaperrr) [inline mods/ra/maps/soviet-11a/soviet11a-AI.lua]: Typo, should be SendCruiser (same for CruiserEntry).

File: mods/ra/maps/soviet-11a/soviet11a-AI.lua [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+PatrolA = { Rocket1, Rocket2 }
+PatrolB = { Rocket3, Rocket4 }
+PatrolC = { Rocket5, Rocket6 }
+PatrolPathA = { RocketPath1.Location, RocketPath2.Location }
+PatrolPathB = { RocketPath3.Location, RocketPath4.Location }
+PatrolPathC = { RocketPath5.Location, RocketPath6.Location }
+
+Helipads = { Helipad1, Helipad2, Helipad3, Helipad4 }
+HeliType = { "heli" }
+DestoryerSquad = { "dd", "dd" }
```
Comment (reaperrr) [inline mods/ra/maps/soviet-11a/soviet11a-AI.lua]: Typo: Should be DestroyerSquad.

File: mods/ra/maps/soviet-11a/soviet11a-AI.lua [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+{
+	easy = 6,
+	normal = 5,
+	hard = 4
+}
+
+AlliedShips =
+{ 
+	easy = { "pt", "pt", "dd" },
+	normal = { "dd", "pt" },
```
Comment (reaperrr) [inline mods/ra/maps/soviet-11a/soviet11a-AI.lua]: It feels a bit weird compared to what we usually do that this squad is weaker on normal than easy, even with the shorter delay.

Maybe make easy 3x pt (or 1 pt + 1 dd), and normal 2x pt + 1x dd?

Comment (Smittytron) [reply mods/ra/maps/soviet-11a/soviet11a-AI.lua]: Yeah I intended 2x dd 1 pt and 3x dd on hard.

File: mods/ra/maps/soviet-11a/soviet11a-AI.lua [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+	easy = 6,
+	normal = 5,
+	hard = 4
+}
+
+AlliedShips =
+{ 
+	easy = { "pt", "pt", "dd" },
+	normal = { "pt", "dd", "dd" },
+	hard = { "dd", "dd" , "dd"}
```
Comment (reaperrr) [inline mods/ra/maps/soviet-11a/soviet11a-AI.lua]: Style nit: there's a spurious space after the 2nd and missing space after the 3rd entry now.

**Comment by Smittytron** [comment]
Updated

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
I think for a mission with such relatively simple design we can count darcyrush's comment as +1, so lgtm.

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/3269d9263953f5dc1ad2ec124dfdfe30362727c6...85416c15b4120079e2c28938203809c7658082dc

**Pull request merged**